front/lib/api/user.ts

```diff
@@ -139,42 +140,44 @@ export async function fetchRevokedWorkspace(
   return new Ok(workspace);
 }
 
-export async function getUserWithWorkspaces(
-  user: UserResource
-): Promise<UserTypeWithWorkspaces> {
+export async function getUserWithWorkspaces<T extends boolean>(
+  user: UserResource,
+  populateExtensionConfig: T = false as T
+): Promise<
```

Comment:
You might want to do this instead:
```suggestion
export async function getUserWithWorkspaces<T extends boolean = false>(
  user: UserResource,
  populateExtensionConfig: T
): Promise<
```

Reply:
Ok I implemented it but then I get this error: https://github.com/dust-tt/dust/actions/runs/12584697848/job/35074872378?pr=9688

<img width="1137" alt="Screenshot 2025-01-02 at 16 31 25" src="https://github.com/user-attachments/assets/60e7459f-6ae4-4687-a49c-ca217a6f9c87" />

=> The suggestion makes the parameter non optional. I'm putting back the previous version 🙇🏻‍♀️